cpmpy/expressions/globalconstraints.py

```diff
@@ -605,5 +630,7 @@ def get_bounds(self):
         """
         Returns the bounds of the (numerical) global constraint
         """
-        arr, val = self.args
-        return 0, len(arr)
+        return [0, len(self.args[0])]
```

Comment:
should this be a tuple?
I liked the previous explicit readable
```
arr, val = self.args
return 0, len(arr)
```

Reply:
this and the repr below are best commited directly on master I think

Reply:
Ah this is a merge slip-up, in master it is correct (we don't need the repr, since it is implemented like the default anyway)